### DISCUSSION TITLE
Avaliable on F Droid

### DISCUSSION THREAD

**Discussion by joanna-murzyn** [anyproto/contributors#253] [Anytype apps]
I noticed that AnyType, despite being open source, is not currently available on F-Droid. 

As a privacy-conscious user who relies on F-Droid for secure app distribution, I'm reaching out to inquire about potential F-Droid inclusion.

**Comment by filip-anytype** [comment]
It's not possible to add Anytype to F-Droid because they only allow apps that are FOSS. You can read more abut our licenses [here](https://legal.any.coop/).

**Reply by visika** [reply]
Maybe it can be added on the IzzyOnDroid F-Droid Repository. This way it would be accessible to anyone adding the repository in F-Droid or using an alternate client like Neo Store.
